src/category_theory/limits/shapes/terminal.lean

```diff
@@ -250,4 +271,52 @@ initial.to _
 
 end comparison
 
+variables {C} {J : Type v} [small_category J]
+
+/--
+If `j` is initial in the index category, then the map `limit.π F j` is an isomorphism.
+-/
+def is_iso_π_of_is_initial {j : J} (I : is_initial j) (F : J ⥤ C) [has_limit F] :
+  is_iso (limit.π F j) :=
+begin
+  refine ⟨⟨_, ⟨_, _⟩⟩⟩,
+  apply limit.lift _
+  { X := F.obj j,
+    π :=
+    { app := λ j', F.map (I.to j'),
+      naturality' := λ j₁ j₂ f,
+      begin
+        dsimp, simp only [category.id_comp, ←F.map_comp, is_initial.to_comp],
+      end, }, },
+  { ext, simp, },
+  { simp, },
+end
```

Comment:
```suggestion
lemma is_iso_π_of_is_initial {j : J} (I : is_initial j) (F : J ⥤ C) [has_limit F] :
  is_iso (limit.π F j) :=
⟨⟨limit.lift _ (cone_of_diagram_initial I F), ⟨by { ext, simp }, by simp⟩⟩⟩
```
and similarly below

Reply:
Thanks. :-) It is so hard to find stuff in the limits API. :-(